src/module/actor/creature/document.ts

```diff
@@ -361,6 +361,20 @@ abstract class CreaturePF2e<
             status.value = Math.min(condition?.value ?? 0, status.max);
         }
 
+        // Add the holy/unholy trait to sanctified spells and actions
+        const traits = this.system.traits.value;
+        const isHoly = traits.includes("holy");
+        const isUnholy = traits.includes("unholy");
+        if ((isHoly || isUnholy) && !(isHoly && isUnholy)) {
+            const sanctifiedItems = [...this.itemTypes.action, ...this.itemTypes.spell].filter((i) =>
```

Comment:
any sanctified feats you might want to spread this to? Dunno if there are any.

Reply:
Ehh, not sure! There aren't any feats with it, but we have such a small sample size.